spec/index.html

```diff
@@ -1814,10 +1814,33 @@ <h2>Entailment rules</h2>
   <p><code>ex:a rdfs:subPropertyOf _:b .<br/>
     _:b rdfs:domain ex:c .<br/>
     ex:d ex:a ex:e .<br/>
-    ex:d _:b ex:c .</code>  by <a>rdfs7</a><br/>
+    ex:d _:b ex:e .</code>  by <a>rdfs7</a><br/>
     <code>ex:d rdf:type ex:c .</code> by <a>rdfs2</a></p>
 
   <p>Where the entailment patterns have been applied to generalized RDF syntax but yield a final conclusion which is legal RDF.</p>
+  
+    <p>
+  The entailment pattern for generalized RDF with [=symmetric RDF triples=],
+  considering that, according to the semantics, the denotation of triple terms should
+  be of type <code>rdfs:Proposition</code>, is the following:
+  </p>
+    <table>
+    <caption>RDFS-T entailment pattern.</caption>
+    <tbody>
+      <tr>
+        <th > </th>
+        <th ><strong>any S</strong></th>
+        <th ><strong>then S RDFS entails</strong></th>
+      </tr>
+      <tr >
+        <td class="othertable"><dfn>Grdfs14</dfn></td>
+        <td class="othertable">if S contains <code>yyy rdf:type rdf:Property .</code><br/>
+          and <code>xxx</code> and <code>yyy</code> appear in E
```

Comment:
This is the only entailment pattern which also depends on E. I think we should either say right above this pattern what the E stands for or we should have a more general pattern here which we slightly adapt when we use it below. 

I am currebtly fighting with git, but will make a proposal once I "won".